From 8f68cdc04892cb3a838386591511fd5b22531e68 Mon Sep 17 00:00:00 2001
From: Bruno Martins <bgcngm@gmail.com>
Date: Tue, 21 Nov 2017 23:28:05 +0000
Subject: [PATCH 02/25] Disable notification channel warnings by default

 * Since we usually ship "userdebug" builds, it is better to globally
   disable the toast because we are not in control of third-party apps
   that haven't been properly aligned with new Oreo requirements.

 * Leave it enabled by default only for "eng" builds

Change-Id: Ie5cdec1cbdd3285af3af730dac4a5de95ad8b334
---
 .../NotificationChannelWarningsPreferenceController.java        | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/com/android/settings/development/NotificationChannelWarningsPreferenceController.java b/src/com/android/settings/development/NotificationChannelWarningsPreferenceController.java
index e451017b3c..7f543d71d2 100644
--- a/src/com/android/settings/development/NotificationChannelWarningsPreferenceController.java
+++ b/src/com/android/settings/development/NotificationChannelWarningsPreferenceController.java
@@ -79,6 +79,6 @@ public class NotificationChannelWarningsPreferenceController extends
 
     @VisibleForTesting
     boolean isDebuggable() {
-        return Build.IS_DEBUGGABLE;
+        return Build.TYPE.equals("eng");
     }
 }
-- 
2.17.1

